modules/document_repository/README.md

```diff
@@ -0,0 +1,30 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a useful tool that provides studies with a centralized location for important documents. Files may be uploaded and organized under any user-defined category. Subcategories can also be defined by the user to be nested under parent categories.
+
+## Intended Users
+ 
+ All Loirs user could view, update, edit and delete the upload files with right permissions.
+
+## Scope
+
+ For all of the Loris users.
```

Comment:
I think the scope should say that it's for study documents, not candidate documents (which go into the media module)